packages/salesforcedx-vscode-core/src/util/orgUtil.ts

```diff
@@ -70,28 +89,27 @@ export async function checkForExpiredOrgs(orgList: OrgList) {
       return;
     }
 
-    const defaultOptions = Aliases.getDefaultOptions();
-    const aliases = await Aliases.create(defaultOptions);
-
-    const formattedOrgsToDisplay = orgsAboutToExpire.map((orgAboutToExpire: any) => {
-      const alias = aliases.getKeysByValue(orgAboutToExpire.username);
-      const aliasName = alias.length > 0
-        ? alias.toString()
+    const formattedOrgsToDisplay = (await Promise.all(orgsAboutToExpire.map(async orgAboutToExpire => {
+      const aliasName = orgAboutToExpire.aliases && orgAboutToExpire.aliases.length > 0
+        ? orgAboutToExpire.aliases[0]
         : orgAboutToExpire.username;
 
+      const authFields = await getAuthFieldsFor(orgAboutToExpire.username);
+
       return nls.localize(
         'pending_org_expiration_expires_on_message',
         aliasName,
-        orgAboutToExpire.expirationDate
+        authFields.expirationDate
       );
-    }).join('\n\n');
+    }))).join('\n\n');
```

Comment:
this formatting code smells like it should be a separate function.  maybe a follow on sepearate ticket.

Reply:
looking at the code, it's been reduced a bit, and I don't think moving the code in the `for()` loop into a separate function is needed now.  FWIW, I have unit tests in packages/salesforcedx-vscode-core/test/vscode-integration/util/orgUtil.test.ts which cover each of the code branches in there ("if isDevHub is true", "if expiration date < today", etc...)